docs/yaml/functions/configure_file.yaml

```diff
@@ -39,6 +39,13 @@ kwargs:
       file objects and more than one file can be passed to the `input` keyword
       argument, see [[custom_target]] for details about string
       substitutions.
+      
+  configuration:
+    type: "[[@cfg_data]] | Dict"
```

Comment:
types don't take markdown `[[@obj]]` references, also there is no such type as `Dict` (but there is `dict`).

Reply:
The type itself is cfg_data, not configuration_data, it just needs to lose the markdown parts (`[[@` and `]]`) because the yaml is already strongly typed so it takes `obj`.

Reply:
Not sure how I missed this, but I guess here we are now... It seems like I even missed that `cfg_data` is returned by the `configuration_data` function, and not a dict...

Also, this should only be `dict[str | bool | int]`.